paper/config/plugins/SimpleAdminHacks/config.yml

```diff
@@ -115,24 +115,28 @@ hacks:
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"All ores except diamond spawn randomly as you dig","color":"green"}]'
+      #- '["",{"text":"[CivMC] ","color":"gold"},{"text":"The message is a lie, there are no essence streaks","color":"green"}]'
+      #- '["",{"text":"[CivMC] ","color":"gold"},{"text":"Lodestones work as elevators, shift to go down and spacebar to go up. They can be made in the Iron Forge factory","color":"green"}]'
+      #- '["",{"text":"[CivMC] ","color":"gold"},{"text":"Do not put buttons or signs over an elevator - or it will be blocked and you will not be able to teleport back to that block!","color":"green"}]'
+      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Consider donating to the server at patreon.com/civ_mc! To get access to some unique features!","color":"blue","clickEvent":{"action":"open_url","value":"https://www.patreon.com/civ_mc"}}]'
+      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"All ores except coal and copper can spawn in big veins. Click for more info","color":"green","clickEvent":{"action":"open_url","value":"https://civwiki.org/wiki/Civ101#HiddenOre"}}]'
+      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"All ores except diamond spawn randomly as you dig. Click for more info","color":"green","clickEvent":{"action":"open_url","value":"https://civwiki.org/wiki/Civ101#HiddenOre"}}]'
       #- '["",{"text":"[CivMC] ","color":"gold"},{"text":"If you do not sleep, the phantoms will get you!","color":"green"}]'
       - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Typing /config will let you adjust various settings","color":"green"}]'
       - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Type /rb to see the crop information for the biome you are in","color":"green"}]'
       - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Type /chunklimits to see the restrictions on how many blocks can be placed in a chunk","color":"green"}]'
       - '["",{"text":"[CivMC] ","color":"gold"},{"text":"If you type /fm you can see all the types of factories you can make","color":"green"}]'
       - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Please review the server rules at reddit.com/r/civmc/wiki/index","color":"green","clickEvent":{"action":"open_url","value":"https://www.reddit.com/r/civmc/wiki/index/"}}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"You may not give another player access to any of your accounts. If you share an account with a friend, you and that friend will be considered the same person.","color":"green"}]'
-      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"Check out the player run newsletter at https://civwiki.news/ to see whats been going on!","color":"green"}]'
-
+      - '["",{"text":"[CivMC] ","color":"gold"},{"text":"The use of VPNs or multiple accounts is not allowed, Logging in from the same place as a friend will ban both of you","color":"green"}]'
```

Comment:
This also can be misconstrued a bit, saying it will ban implies it's against the rules as well as vpns and multiple accounts.